modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This document helps you use Object Store v2 to store and view data.
+Use this tutorial to get started with using Object Store v2 to store and view data.
```

Comment:
We could avoid two "use"s by something like "This tutorial helps you to start using ...."